okta/utils.go

```diff
@@ -238,7 +241,10 @@ func ensureNotDefault(d *schema.ResourceData, t string) error {
 
 func getMapString(m map[string]interface{}, key string) string {
 	if v, ok := m[key]; ok {
-		return v.(string)
+		if res, ok := v.(string); ok {
+			return res
+		}
+		return ""
```

Comment:
extra there's an extra `return ""`, it would occur on 249 when coming out of the if statement